README.md

```diff
@@ -1,5 +1,4 @@
 # Command Palette package
-[![CI](https://github.com/atom/command-palette/actions/workflows/ci.yml/badge.svg)](https://github.com/atom/command-palette/actions/workflows/ci.yml)
 
 Find and run available commands using <kbd>cmd-shift-p</kbd> (macOS) or <kbd>ctrl-shift-p</kbd> (Linux/Windows) in Atom.
```

Comment:
```suggestion
Find and run available commands using <kbd>cmd-shift-p</kbd> (macOS) or <kbd>ctrl-shift-p</kbd> (Linux/Windows) in Pulsar.
```

One last change then this will be good to merge